youtube_dl/extractor/facebook.py

```diff
@@ -806,6 +807,9 @@ def _resolve_thumbnail(self, webpage, tahoe_data):
 
```

Comment:
please remove the twitter:image from here as it has nothing to do with twitter ;-)